src/table/TableBodyWrapper.jsx

```diff
@@ -126,8 +129,13 @@ TableBodyWrapper.propTypes = {
   selectionsAPI: PropTypes.object.isRequired,
   layout: PropTypes.object.isRequired,
   theme: PropTypes.object.isRequired,
-  focusedCellCoord: PropTypes.object.isRequired,
+  focusedCellCoord: PropTypes.arrayOf(PropTypes.number),
+  handleFocusedCellCordsUpd: PropTypes.func.isRequired,
   setShouldRefocus: PropTypes.func.isRequired,
+  handleSetMovingTop: PropTypes.func.isRequired,
 };
 
-export default React.memo(TableBodyWrapper);
+export default memo(TableBodyWrapper, (prevProps, nextProps) => {
+  if (prevProps.focusedCellCoord.toString() !== nextProps.focusedCellCoord.toString()) return true;
+  else return false;
+});
```

Comment:
```suggestion
export default memo(TableBodyWrapper, (prevProps, nextProps) => prevProps.focusedCellCoord.toString() !== nextProps.focusedCellCoord.toString());
```

Reply:
So this does not update the component, when focusCellCoord has changed? Could there be cases when we want to update when it has changed? When testing it, it seems to work, but we need more manual testing for sure.

Reply:
Yes, when `focusCellCoord` state changes, we are not going to re-render our `<TableHead />` and `<TableBody />` component. Since we don't actually need the value of it and the only thing we need inside of these component is the setter function for that state which has been used inside of some helper function like `headHandleKeyPress`